description: 'drm: Fix use-after-free read in drm_getunique()'
references:
- https://www.cve.org/CVERecord?id=CVE-2021-47280
- https://git.kernel.org/stable/c/17dab9326ff263c62dab1dbac4492e2938a049e4
- https://git.kernel.org/stable/c/7d233ba700ceb593905ea82b42dadb4ec8ef85e9
- https://git.kernel.org/stable/c/b246b4c70c1250e7814f409b243000f9c0bf79a3
- https://git.kernel.org/stable/c/491d52e0078860b33b6c14f0a7ac74ca1b603bd6
- https://git.kernel.org/stable/c/f773f8cccac13c7e7bbd9182e7996c727742488e
- https://git.kernel.org/stable/c/b436acd1cf7fac0ba987abd22955d98025c80c2b
- https://lore.kernel.org/linux-cve-announce/2024052153-CVE-2021-47280-4344@gregkh/
- https://git.kernel.org/linus/b436acd1cf7fac0ba987abd22955d98025c80c2b
comments:
  cip/cip-kernel-sec: Introduced commit is not determined.Fixed in v5.13-rc6.
  cip/masami: It seems to be Linux 4.4 is affected.
  debian/carnil: First introducing commit could not be determined.
fixed-by:
  cip/4.19: [7d233ba700ceb593905ea82b42dadb4ec8ef85e9]
  cip/4.19-rt: [7d233ba700ceb593905ea82b42dadb4ec8ef85e9]
  cip/5.10: [491d52e0078860b33b6c14f0a7ac74ca1b603bd6]
  cip/5.10-rt: [491d52e0078860b33b6c14f0a7ac74ca1b603bd6]
  debian/buster-security: ['version:debian/4.19.208-1']
  mainline: [b436acd1cf7fac0ba987abd22955d98025c80c2b]
  openela/4.14: [17dab9326ff263c62dab1dbac4492e2938a049e4]
  stable/4.14: [17dab9326ff263c62dab1dbac4492e2938a049e4]
  stable/4.19: [7d233ba700ceb593905ea82b42dadb4ec8ef85e9]
  stable/5.10: [491d52e0078860b33b6c14f0a7ac74ca1b603bd6]
  stable/5.4: [b246b4c70c1250e7814f409b243000f9c0bf79a3]
  ubuntu/bionic: [894bc4bdbb11642181b70efcbbf526149e82331b]
  ubuntu/focal: [b2b00399507789525ca2962ea7a7c65619ec672d]
  ubuntu/groovy: [344bafe8647252cd5f70c5278c391d06f7189bcc]
